JASP-Engine/JASP/R/selectionmodels.R

```diff
@@ -271,18 +271,18 @@ SelectionModels <- function(jaspResults, dataset, options, state = NULL) {
   if (!is.null(fit)) {
     if (!class(fit) %in% c("simpleError","error")) {
       rowREUnadjustedTau  <- c(rowREUnadjustedTau, list(
-        est  = sqrt(fit[["unadj_est"]][1,1]),
-        stat = fit[["z_unadj"]][1,1],
-        pval = fit[["p_unadj"]][1,1],
-        lCI  = sqrt(ifelse(fit[["ci.lb_unadj"]][1,1] < 0, 0, fit[["ci.lb_unadj"]][1,1])),
-        uCI  = sqrt(fit[["ci.ub_unadj"]][1,1])
+        est  = sqrt(fit[["unadj_est"]][1, 1]),
+        stat = fit[["z_unadj"]][1, 1],
+        pval = fit[["p_unadj"]][1, 1],
```

Comment:
perhaps pValue or pVal might be more descriptive

Reply:
changed to pVal